.gitignore

```diff
@@ -1 +1,9 @@
-_build
\ No newline at end of file
+_build

```

Comment:
Do we really need this folder? I think that addition was not centralized one. Seems like nobody cares today.

Reply:
It's seems to be added in scope of some doc build updates. @dcbenito, may be you can quickly confirm if it is still actual

Reply:
Seems like it's for documentation purpose but seems very specific

Reply:
We need to ignore the _build folder because that is what I use when building the documentation for my testing/uploading to IDZ. Please keep it.

Reply:
I am wondering if we have some specific folder name for documentation building somewhere in our documentation. If yes, I would suggest that everybody uses it consistently and we should add it in .gitignore. If not, please add comment to this line why we need that.

Reply:
Having a "build" folder is standard across most GitHub doc projects I've seen/been involved with. I don't see what needs to change.